interface-elements-for-web/articles/gantt/toolbar.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Toolbar
+author: Anna Kondratova
+---
+
+# Toolbar
+
+![Gantt - toolbar](~/interface-elements-for-web/images/Gantt/toolbar.png)
+
+The toolbar provides the following commands:
+
+| Command | Description |
+|---|---|
+| Undo | Reverses the last action. |
+| Redo | Repeats the undone action. |
+| Add Task | Adds a task. |
+| Delete Task | Deletes a task. |
+| Collapse Tasks  | Collapses all tasks. |
+| Expand Tasks | Expands all tasks. |
+| Zoom In | Zooms in the chart. |
+| Zoom In and Zoom Out | Zooms out the chart. |
```

Comment:
```suggestion
| Zoom In and Zoom Out | Zooms out of the chart. |
```

Reply:
Yes, I've missed this mistake.

Reply:
updated to Zoom Out

Reply:
Thank you. Also, change "Zooms out the chart." to "Zooms out of the chart."